app/javascript/gobierto_visualizations/webapp/containers/contracts/ContractsShow.vue

```diff
@@ -1,68 +1,120 @@
+          :value="contractor"
+          :icon="'building'"
+        />
+      </div>
+      <ContractsShowLabelHeader
+        :label="labelStatus"
+        :value="status"
+        :icon="'columns'"
+      />
+      <ContractsShowLabelHeader
+        :label="labelType"
+        :value="contract_type"
+        :icon="'clipboard-list'"
+      />
+      <ContractsShowLabelHeader
+        :label="labelProcess"
+        :value="process_type"
+        :icon="'archive'"
+      />
+      <div class="pure-u-1 pure-u-lg-1-1">
+        <ContractsShowLabelHeader
+          :label="labelCategory"
+          :value="category_title"
+          :icon="'tag'"
+        />
+      </div>
+      <div class="visualizations-contracts-show__body">
+        <div
+          v-show="minorContract"
+          class="pure-u-1 pure-u-lg-1-2"
+        >
+          <div class="pure-u-1 pure-u-lg-1-1 visualizations-contracts-show__body__group">
+            <span class="visualizations-contracts-show__text__header">{{ labelTender }}</span>
+            <span class="visualizations-contracts-show__text">{{ parseDate(submission_date) }}</span>
+            <i
+              v-show="showArrowDate"
+              class="fas fa-arrow-right"
+            />
+            <span class="visualizations-contracts-show__text">{{ parseDate(open_proposals_date) }}</span>
+          </div>
+          <ContractsShowLabelGroup
+            :label="labelBidDescription"
+            :value="initial_amount_no_taxes | money"
+          />
+          <ContractsShowLabelGroup
+            v-show="numberOfProposals"
+            :label="labelBiddersDescription"
+            :value="numberOfProposals"
+          />
+        </div>
+        <div class="pure-u-1 pure-u-lg-1-2">
+          <ContractsShowLabelGroup
+            :label="labelAwarding"
+            :value="parseDate(award_date)"
+          />
+          <ContractsShowLabelGroup
+            :label="labelContractAmount"
+            :value="final_amount_no_taxes | money"
+          />
+          <template v-if="!hasBatch">
+            <div class="pure-u-1 pure-u-lg-1-1 visualizations-contracts-show__body__group">
+              <span class="visualizations-contracts-show__text__header">{{ labelAssigneeDescription }}</span>
+              <router-link
+                id="assignee_show_link"
+                :to="{ name: 'assignees_show', params: {id: assignee_routing_id } }"
+              >
+                <strong class="d_block">{{ assignee }}</strong>
+              </router-link>
+            </div>
+          </template>
+          <template v-else>
+            <ContractsShowTable
+              :data="filterContractsBatchs"
+            />
+          </template>
         </div>
       </div>
-
       <div class="pure-u-1 pure-u-lg-1-2">
-        <table>
+        <span class="visualizations-contracts-show__text__header">{{ labelQuestionDescription }}</span>
+        <table class="visualizations-contracts-show-table">
```

Comment:
Cannot this table be a `ContractsShowTable` component?

Reply:
You're right. I've divided it into two components